src/js/icons/index.d.ts

```diff
@@ -493,6 +495,7 @@ export declare const StackOverflow: Icon;
 export declare const Stakeholder: Icon;
 export declare const StarHalf: Icon;
 export declare const Star: Icon;
+export declare const StarUnfull: Icon;
```

Comment:
```suggestion
export declare const StarUnfill: Icon;
```
Also, I'm wondering if this should be called `StarOutline` instead of `Unfill`?

Reply:
oh yeah good call